ui/rewards/RewardsLoginModal.tsx

```diff
@@ -0,0 +1,57 @@
+import { Modal, ModalOverlay, ModalContent, ModalHeader, ModalCloseButton, ModalBody, useBoolean } from '@chakra-ui/react';
+import React, { useCallback, useEffect } from 'react';
+
+import { useRewardsContext } from 'lib/contexts/rewards';
+import useIsMobile from 'lib/hooks/useIsMobile';
+import useWallet from 'lib/web3/useWallet';
+
+import CongratsStepContent from './steps/CongratsStepContent';
+import LoginStepContent from './steps/LoginStepContent';
+
+const RewardsLoginModal = () => {
```

Comment:
Can we put "dashboard" components and "login modal" components into separate sub-directories?